app/services/trainees/create_from_hesa.rb

```diff
@@ -4,15 +4,19 @@ module Trainees
   class CreateFromHesa
     include ServicePattern
 
+    USERNAME = "HESA"
+
     def initialize(itt_record_doc:)
       @hesa_trainee = Hesa::Parsers::IttRecord.to_attributes(itt_record_doc: itt_record_doc)
       @trainee = Trainee.find_or_initialize_by(hesa_id: hesa_trainee[:hesa_id])
     end
 
     def call
-      trainee.assign_attributes(mapped_attributes)
-      trainee.save!
-      trainee
+      Audited.audit_class.as_user(USERNAME) do
```

Comment:
Suggest putting a comment here to remind us later to move the `Audited.audit_class.as_user` into the Hesa job when it's created.